backend/pom.xml

```diff
@@ -10,7 +10,7 @@
     </parent>
     <groupId>ca.bc.gov.hlth</groupId>
     <artifactId>moh-ums</artifactId>
-    <version>0.0.2-SNAPSHOT</version>
+    <version>0.0.1-SNAPSHOT</version>
```

Comment:
Not sure what to do about this. Maybe nothing, just fix it in `master`.

Reply:
This is what we agreed to do in the chat; leave it as 0.0.1-SNAPSHOT and just use GitHub's Releases/Tags for identifying the actual version in Prod. I think I previously had moved it from 0.0.2-SNAPSHOT to 1.3.0 or something to reflect what we were at currently but then we agreed not to use versioning in Maven at all.